client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -384,6 +395,21 @@ export const SeoForm = React.createClass( {
 		this.setState( { showPreview: false } );
 	},
 
+	getConflictingSeoPlugins( activePlugins ) {
+		const conflictingSeoPlugins = [
+			'Yoast SEO',
+			'Yoast SEO Premium',
+			'All In One SEO Pack',
+			'All in One SEO Pack Pro',
+		];
+
+		const plugins = flatMap( activePlugins, plugin => (
```

Comment:
Unneeded leftover, I've just noticed after filling the PR and was about to correct it but you beat me to it. :smile: